src/cli.js

```diff
@@ -29,6 +29,10 @@ program
     '--open [browser]',
     'automatically open in specified browser, defaults to default browser'
   )
+  .option(
+    '--spawn [command]',
+    'automatically spawns a child process inheriting stdio, after bundle is ready'
```

Comment:
Yes, I originally used stdio inheriting - but actually moved to parse each line so that I can prefix (e.g. make host process logs and child process logs more easily distinguishable).

Reply:
Removed the remarks from cli help not to mention "inheriting".